RHEL/7/input/profiles/docker-host.xml

```diff
@@ -1,5 +1,5 @@
-<Profile id="rhel7-docker">
-<title>Standard Docker Security Profile for RHEL7</title>
+<Profile id="docker-host">
+<title>Standard Docker Host Security Profile for RHEL7</title>
```

Comment:
Just nitpick. Can we change this to "Red Hat Enterprise Linux 7"? I don't know anything about "RHEL7" in official presentation :).

Otherwise looks good to me.